### ISSUE TITLE
[BUG] Core don't work after waiting in Windows

### ISSUE THREAD

**Issue by Winfidonarleyan** [azerothcore/azerothcore-wotlk#3186]
Edit: only affecting:

```
Microsoft Visual Studio Community 2019 Preview
Ver. 16.7.0 Preview 3.1
```

Works fine when using VS version `16.6.3`

##### CURRENT BEHAVIOUR:
- Сore stops working after ~10 minutes of waiting
- No crash logs
![image](https://user-images.githubusercontent.com/18329778/87299442-39644580-c536-11ea-9a49-1919edd548a5.png)

##### STEPS TO REPRODUCE THE PROBLEM:
1. Start `worldserver.exe`
2. Wait ~10 min
3. Try to write `.ser info`

##### EXTRA NOTES:
- 

##### OPERATING SYSTEM:
- Win 10 x64 1909
```
Microsoft Visual Studio Community 2019 Preview
Ver. 16.7.0 Preview 3.1
```

##### MODULES:
- [mod-transmog](https://github.com/azerothcore/mod-transmog)

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan this looks like a serious bug and it's probably introduced by any of the recent changes.

Can you go back a bit in history and locate which commit has caused the issue?

**Comment by Winfidonarleyan** [comment]
Maybe this a12e58b Need more tests

**Comment by Kitzunu** [comment]
On https://github.com/Azerothcore/azerothcore-wotlk/commit/20c7da238b17d32575d7ef8eca1018c20acdd7eb I can't reproduce 
```
Server uptime: 13 minute(s) 16 second(s).
```
Will try with latest master

**Comment by Kitzunu** [comment]
On e359cb008578 (Latest master) I cannot reproduce this issue.
```
Server uptime: 13 minute(s) 46 second(s).
```

**Comment by BarbzYHOOL** [comment]
shutdown the server if idle (and no player on) is a gm command

**Comment by Winfidonarleyan** [comment]
Maybe I found a problem

**Comment by Winfidonarleyan** [comment]
Okay, good, very good
![image](https://user-images.githubusercontent.com/18329778/87383486-880bf100-c5c3-11ea-8e98-c2de03877383.png)
Compile witch 
```
Microsoft Visual Studio Community 2019
Ver. 16.6.3
```
Bad compile in 
```
Microsoft Visual Studio Community 2019 Preview
Ver. 16.7.0 Preview 3.1
```

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan then we have to:

- write in the wiki to use ` 16.6.3` until this bug is fixed
- keep this open until it's fixed in `16.7.0` too

**Comment by Kitzunu** [comment]
Will close this as it works on 16.8.3, as it is a newer version
